x-pack/filebeat/module/checkpoint/firewall/ingest/pipeline.yml

```diff
@@ -471,6 +475,10 @@ processors:
         } else if (iana_number == '132') {
             ctx.network.transport = 'sctp';
         }
+  - convert:
+      field: checkpoint.packets
+      type: long
+      ignore_missing: true
   - rename:
       field: checkpoint.packets
       target_field: network.packets
```

Comment:
Can this be merged into a single step - https://github.com/elastic/integrations/pull/3800#discussion_r927482597

Reply:
Same as above.